src/modm/platform/gpio/stm32/config.hpp.in

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2016-2018, Niklas Hauser
+ *
+ * This file is part of the modm project.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+// ----------------------------------------------------------------------------
+
+#pragma once
+
+enum class
+Config : uint32_t
+{
+	/// @cond
+	_Mode        = 0x8000'0000,
```

Comment:
Identifiers with `_` + upper case letters are reserved in C++ for the standard library implementation.